src/PayID/pay-id-client.ts

```diff
@@ -89,8 +89,9 @@ export default class PayIDClient {
         throw new PayIDError(PayIDErrorType.UnexpectedResponse, message)
       }
       // TODO(keefertaylor): make sure the header matches the request.
-    } else if (data?.addressDetails) {
-      return data.addressDetails
+    } else if (data?.addresses) {
+      // TODO(amiecorso): what if addresses is empty or contains more than one CryptoAddressDetails?
```

Comment:
Given that this client only ever makes a request for one type of address (right @dino-rodriguez ?) I think we should throw an error here.